test/index.spec.ts

```diff
@@ -56,3 +53,55 @@ describe('Test createUser', () => {
     await User.delete({ email: userInput.email });
   });
 });
+
+describe('Test createUser with existing email', () => {
+  it('should return an error for trying to create a user with email already registered', async () => {
+    const query = `mutation($input: UserInput!) {
+      createUser(input: $input) {
+        id, name, email, birthdate
+      }
+    }`;
+
+    const userInput = {
+      name: 'Teste',
+      email: 'teste2@email.com',
+      password: 'senha123',
+      birthdate: '01-01-2000',
+    };
+
+    const result = await connection.post('/graphql', { query, variables: { input: userInput } });
+
+    expect(result.data.errors).to.be.deep.eq([
+      {
+        message: 'Email already registered.',
+        code: 401,
+      },
+    ]);
+  });
+});
+
+describe('Test createUser with invalid password', () => {
+  it('should return an error for trying to create a user with a password that does not meet the requirements', async () => {
+    const query = `mutation($input: UserInput!) {
```

Comment:
Fica um pouco de overhead se a a gente tratar cada caso de teste de cada quey/mutation num describe diferente. O describe do mocha é feito justamente para agrupar testes. E não é só um agrupamento que fica bonitinho na hora de mostrar o relatório de testes, mas também um agrupamento que te permite ter um escopo em comum entre um grupo de testes, em que vc pode reaproveitar variáveis, se for uma boa ideia.

Talvez [esse doc](https://www.notion.so/taqtile/Boas-pr-ticas-para-testes-66f8b1e317be4d88bde55604d45f8fe1) te ajude um pouco com algumas boas práticas. A nossa sugestão é usar um describe para cada query ou mutation, e todos os casos de teste dela dentro desse describe.